index.html

```diff
@@ -3,134 +3,401 @@
+       </select>
     </div>
   </div>
 </div>
 <div class="main section">
-  <div class="container">
+  <div class="col-sm-6">
     <div id="chart"></div>
   </div>
+  <div class="col-sm-6">
+    <div id="chart2"></div>
+  </div>
 </div>
 <script type="text/javascript" src="https://npmcdn.com/d3@3.5.17/d3.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/science@1.9.3/science.v1.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    state: 'all',
+    region: 'alll',
   };
-
-  var svg;
-
+  var svg, x, y, svg2, x2, y2;
+  var margin = { top: 15, right: 15, bottom: 40, left: 80 };
+  var width = 700 - margin.right - margin.left;
+  var height = 500 - margin.top - margin.bottom;
   // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+  
 
-  function setup() {
-    // SVG
+  function setup() {    
 
+    // SVG 1
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
-    // SCALES
+    //SVG 2
+     svg2 = d3.select('#chart2')
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
+    // SCALES 1
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.PovertyRate; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+    rscale = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) {return d.Pop; })])
+      .range([2,20]);
+
+     // SCALES 2
+    x2 = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.MedianIncome; })])
+      .range([0, width])
+      .nice();
+    y2= d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+     
 
     // AXES
+    var padding=0
+    var axispadding=60
+    var xaxispadding=35
 
-    // EVENT HANDLERS
 
+    //Chart 1 Axes
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .orient("bottom")
+      .ticks(10);
+
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .orient("left")
+      .ticks(10);
+    
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate(0,"+ height + ")")
+      .call(xAxis);
+
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate("+ padding + ",0)")
```

Comment:
Padding is sometimes used as a replacement for the margin pattern. I see that you've set it to 0, effectively disabling it. We can probably go one step further and remove this entirely (the padding for your axis labels is absolutely a valid and clear way to do that, though).